doc/user_manual/sampler.tex

```diff
@@ -2319,6 +2404,95 @@ \subsubsection{Limit Surface Search}
     return -1
 \end{lstlisting}
 
+%%% Adaptive Samplers: ADMC
+\subsubsection{Adaptive Monte Carlo}
+\label{subsubsubsec:ADMC}
+The \textbf{Adaptive Monte Carlo} approach is an advanced methodology
+employing a smart sampling around .....
+%
+The main idea of the application of the previously explained adaptive sampling
+approach to ......
+
+This process continues until the standard error of all the metrics is less than the specified tolerance.
+%
+
+\specBlock{an}{AdaptiveMonteCarlo}
+%
+\begin{itemize}
+  \itemsep0em
+  \item \nameDescription
+\end{itemize}
+
+\variableIntro{AdaptiveMonteCarlo}
+\begin{itemize}
+\item \variableDescription
+  \variableChildrenIntro
+  \begin{itemize}
+    \item \distributionDescription
+    \item \functionDescription
+  \end{itemize}
+  \item \constantVariablesDescription
+\end{itemize}
+
+ In addition to the \xmlNode{variable} nodes, the main
+\xmlNode{AdaptiveMonteCarlo} node needs to contain two supplementary
+sub-nodes:
+
+\begin{itemize}
+  \item \convergenceDescriptionAMC
+  % Assembler Objects
+  \item \assemblerDescription{AdaptiveMonteCarlo}
+    \begin{itemize}
+      \item \xmlNode{initialSeed}, \xmlDesc{integer, optional field},
+      initial seeding of random number generator for Monte Carlo sampler.
+
+      \item \xmlNode{TargetEvaluation}, \xmlDesc{string, required field},
+        represents the container where the system evaluations are stored.
+        %
+        From a practical point of view, this XML node must contain the name of
+        a data object defined in the \xmlNode{DataObjects} block (see
+        Section~\ref{sec:DataObjects}).
+        %
+        The adaptive sampling accepts ``DataObjects'' of type
+        ``PointSet'' only.
```

Comment:
open an issue to extend this to time-dependent... We can find a way to allow convergence on the history.